src/Header.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
same notes as in the Calendar component. Please move it to the separate folder and avoid 'any' type

Reply:
moved, and comment as in Calendar